app/routes/login.js

```diff
@@ -6,7 +6,8 @@ const { sendFarmerApplyLoginMagicLink } = require('../lib/email/send-magic-link-
 const { clear } = require('../session')
 const { sendMonitoringEvent } = require('../event')
 
-const hintText = 'We\'ll use this to send you a link to apply for a review'
+const hintText = 'We\'ll use this to send you a link to apply for a review. This must be the business email address linked to the business applying for a review.'
```

Comment:
instead of using ' 
for consistency you could use ’ like in below:

<img width="458" alt="image" src="https://user-images.githubusercontent.com/118533775/213434956-fec41dce-bf16-4f2d-b8d4-b0ecf1d63777.png">

Reply:
Content designers determine this.

Reply:
instead of: We\'ll use this...

please use: We’ll use this...

’ instead of '